src/main/scala/io/iohk/ethereum/jsonrpc/JsonMethodsImplicits.scala

```diff
@@ -38,7 +38,7 @@ trait JsonMethodsImplicits {
     }.toEither.left.map(_ => InvalidParams())
 
   private def decode(s: String): Either[JsonRpcError, Array[Byte]] = {
-    if(!s.startsWith("0x")) Left(InvalidParams())
+    if(!s.isEmpty && !s.startsWith("0x")) Left(InvalidParams())
```

Comment:
@ntallar that request works for me. Using geth-classic `Geth/v3.3.0-1-gdd95f05/darwin/go1.7`.

Either way being more permissive (allowing unprefixed address for consistency) doesn't hurt in this case.